packages/paste-website/src/pages/getting-started/design.mdx

```diff
@@ -62,11 +62,11 @@ If you’re unfamiliar with Abstract, version control, or git workflows, read th
 
 - *Merge*: The way you combine your branch with the Master files
 
-- *Merge conflict*: This happens when there’s a conflicting version of an artboard with the Master and a branch. Abstract will walk you through which artboards are conflicting so you can choose what want to appear in Master.
+- *Merge conflict*: This happens when there’s a conflicting version of an artboard with the Master and a branch. Abstract will walk you through which artboards are conflicting so you can choose what you want to appear in Master. The best way to avoid these conflicts would be to plan out and split up the work between your collaborators before making the changes. You can also split up your Sketch files into more granular pieces (*e.g.*, split a Sketch library and product feature work into separate files). Merge conflicts can prevent you from overriding work unintentionally, but can’t tell you which change is the “right” one.
```

Comment:
Consider making these ways a list. So "To avoid these conflicts, we recommend: * plan * something else"